lib/js/index.js

```diff
@@ -126,12 +126,21 @@ class DateTimePicker extends Events {
     this.$('.js-ok')
       .addEventListener('click', () => this.clickSubmit(), false);
 
+    this.$('.c-datepicker__header-date__hours').addEventListener('click', e => this.showHourClock(e), false);
```

Comment:
Please use `.js-date-hours` here - BEM classnames are for style, `js-` classes are for JavaScript. I know we don't totally adhere to this yet, but that's the aim 😄 

The idea comes from [this article](https://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/) by Harry Roberts, btw - it's a good read!